packages/features/ee/managed-event-types/reassignment/lib/cancelWorkflowReminders.ts

```diff
@@ -0,0 +1,27 @@
+import { WorkflowRepository } from "@calcom/features/ee/workflows/repositories/WorkflowRepository";
+import logger from "@calcom/lib/logger";
+import type { WorkflowMethods } from "@calcom/prisma/enums";
+
+const workflowLogger = logger.getSubLogger({ prefix: ["managedEventWorkflowsCancellation"] });
+
+export async function cancelWorkflowRemindersForReassignment({
```

Comment:
why do we need this function? why can't we call   `await WorkflowRepository.deleteAllWorkflowReminders(workflowReminders);` directly?

Reply:
Looking at this again, this was indeed not left in the refactor. This is intentional in the sense that it bundles the repository call with the workflow-specific logging/telemetry contract, keeping higher-level code clean and making future changes localized.
We could remove this and call `await WorkflowRepository.deleteAllWorkflowReminders(workflowReminders);` directly, but I think this is helpful for debugging down the line and worth it.